client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/BackfilaWebApiActions.kt

```diff
@@ -27,6 +28,7 @@ internal class PrepareBackfillAction @Inject constructor(
   @ResponseContentType(MediaTypes.APPLICATION_PROTOBUF)
   @Authenticated(services = ["backfila"])
   @LogRequestResponse(bodySampling = 1.0, errorBodySampling = 1.0)
+  @AvailableWhenDegraded
```

Comment:
I thought this annotation is supposed to go on the server-side

Reply:
This is the API that gets created by the client that backfila uses to call back in.

Reply:
nvm, this is the server...